test/fiberv2/v2.52.5/go.mod

```diff
@@ -0,0 +1,32 @@
+module fiberTest/v2.52.5
```

Comment:
The directory should be named with the lowest version which the rule supports. It should be v2.43.0 here

Reply:
And the dependency in `go.mod` should also be updated